js/main.js

```diff
@@ -0,0 +1,20 @@
+const hamburger = document.querySelector('.hamburger');
+const openMenu = document.querySelector('.mobile-option');
+const extBtn = document.querySelector('.exit');
+
+function show() {
+  openMenu.style.display = 'block';
+  openMenu.classList.toggle('open');
+  hamburger.style.visibility = 'hidden';
+  extBtn.style.visibility = 'visible';
+}
+
+function close() {
+  openMenu.style.display = 'none';
+  openMenu.classList.remove('open');
+  hamburger.style.visibility = 'visible';
+  extBtn.style.visibility = 'hidden';
+}
+
```

Comment:
- [ ] You have not implemented all the changes requested by the previous reviewer. Kindly implement them. Whenever a user clicks on any of the top navigation links (portfolio, about or contact) two things should happen:
    1. The link should take the user to the clicked section on the main page. (this is working correctly) ✅ 
    2. The navigation pop-up should close just like when you click the x button. (this is not happening) ❌
   Look at these two images, one of them is your implementation and the second shows the correct implementation.

    | Your implementation | Correct Implementation |
    |----------------------|-------------------------|
    |![nurka mobile](https://user-images.githubusercontent.com/62938735/189859682-2f664b5b-2720-4c45-a651-8edea284040a.gif)| ![correct mobile](https://user-images.githubusercontent.com/62938735/189859888-c61c371c-87a5-4f97-baa1-2373e4decdfe.gif)|

One way to achieve this is to add an event listener to the buttons that call your close function `onclick`